react-app-rewired/index.ts

```diff
@@ -58,6 +58,7 @@ const reactAppRewireConfig = {
 
     // Initialize top-level arrays just in case they're missing for some reason.
     _.merge(config, {
+      devtool: isProduction ? 'source-map' : 'inline-source-map',
```

Comment:
I can't confirm whether this actually works or not. I can remove it if it doesn't provide value.